pgmpy/tests/test_continuous/test_discretizers.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+
+import numpy as np
+import numpy.testing as np_test
+from scipy.stats import gamma, expon
+
+from pgmpy.factors import ContinuousFactor
+from pgmpy.continuous.discretize import BaseDiscretizer
+from pgmpy.continuous.discretize import RoundingDiscretizer
+from pgmpy.continuous.discretize import UnbiasedDiscretizer
+
+
+# class TestBaseDiscretizer(unittest.TestCase):
```

Comment:
@khalibartan I need to either change them or remove them. The work is still
in progress as there are few things that need to be changed in this PR.
On 2 Jun 2016 12:18, "Utkarsh" <notifications@github.com> wrote:

> In pgmpy/tests/test_continuous/test_discretizers.py
> <https://github.com/pgmpy/pgmpy/pull/686#discussion_r65491528>:
>
> > @@ -0,0 +1,153 @@
> > +import unittest
> > +
> > +import numpy as np
> > +import numpy.testing as np_test
> > +from scipy.stats import gamma, expon
> > +
> > +from pgmpy.factors import ContinuousFactor
> > +from pgmpy.continuous.discretize import BaseDiscretizer
> > +from pgmpy.continuous.discretize import RoundingDiscretizer
> > +from pgmpy.continuous.discretize import UnbiasedDiscretizer
> > +
> > +
> > +# class TestBaseDiscretizer(unittest.TestCase):
>
> @yashu-seth <https://github.com/yashu-seth> Why is this section commented
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pgmpy/pgmpy/pull/686/files/c5e1d428a628817c88267c739c9313944826e841#r65491528>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANikoR3wpHuTeN955AwO835qrEW7BVJOks5qHnzGgaJpZM4IZogz>
> .
>